packages/airnode-deployer/terraform/state/gcp/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "gcp_project" {
+  description = "GCP project for deployment"
+}
+
+variable "gcp_region" {
+  description = "GCP region for deployment"
+  default     = "us-east1"
```

Comment:
Not really sure it would be useful for our use case. Neither cloud functions nor buckets support zones (or rather they are spread through multiple zones by default).